### ISSUE TITLE
Explanation for the current minSdkVersion

### ISSUE THREAD

**Issue by Johnson145** [ranaparamveer/android-sqlite-asset-helper#4]
Can you explain why you've increased the required `minSdkVersion` to 19 ([see this commit](https://github.com/ranaparamveer/android-sqlite-asset-helper/commit/aa04a0178e91bd33ef4f2d892ecbea741c607af5#diff-c197962302397baf3a4cc36463dce5ea))?

I'm just about to switch from the original repo to your fork, but noticed that it got incompatible with older versions. In my case, I'm trying to support minSdkVersion=16, too. Just wondering whether there is an actual incompatibility?

Update:
Some initial tests on SDK 18 seem to work without any issues.

**Comment by Kisty** [comment]
I agree there's no reason for `minSdkVersion` to be 19. Why not make a PR to bring it down to `8` as that's what it was before.
